unsafe.go

```diff
@@ -32,7 +32,7 @@ func unsafeByteSlice(base unsafe.Pointer, offset uintptr, i, j int) []byte {
 // manipulation of reflect.SliceHeader to prevent misuse, namely, converting
 // from reflect.SliceHeader to a Go slice type.
 func unsafeSlice(slice, data unsafe.Pointer, len int) {
-	s := (*reflect.SliceHeader)(slice)
+	s := (*reflect.SliceHeader)(slice) //nolint:staticcheck
```

Comment:
I think we need to backport https://github.com/etcd-io/bbolt/pull/566 to release-1.3.
- Remove `UnsafeSlice`, and use `unsafe.Slice`

It's OK to fix it in a followup

Reply:
Will do the porting in another PR!

Reply:
Porting is attempted at https://github.com/etcd-io/bbolt/pull/717